### PR TITLE
feat(alert): Connect wizard options to alert rule builder

### PR DESCRIPTION
**Pull request by davidenwang**
Using a similar pattern to `createFromDiscover` I have predefined alert types defined for each alert wizard option which will populate three query args:
- `aggregate`
- `datasource`
- `eventTypes`

these three fields are used to pre-fill the unsaved metric rule. 

Selecting `Issue Alert` on the wizard will set `createFromWizard=true`, but will not provide any of the above three fields, and the builder will just show the user the blank issue alert builder.

Kapture 2021-03-23 at 18 17 36

### CODE REVIEW

File: src/sentry/static/sentry/app/views/alerts/wizard/index.tsx [hunk 1]
```diff
@@ -39,6 +44,27 @@ class AlertWizard extends React.Component<Props, State> {
     this.setState({alertOption});
   };
 
+  renderCreateAlertButton = () => {
```
Comment (scttcper) [inline src/sentry/static/sentry/app/views/alerts/wizard/index.tsx:47]: doesn't need to be an arrow function

File: src/sentry/static/sentry/app/views/alerts/wizard/options.tsx [hunk 2]
```diff
@@ -39,3 +40,37 @@ export const AlertWizardDescriptions: Record<AlertType, string> = {
...
   lcp: t('Alert on longest contentful paint'),
 };
+
+export type WizardRuleTemplate = {
+  aggregate: string;
+  dataset: Dataset;
+  eventTypes: EventTypes;
+};
+
+export const AlertWizardRuleTemplates: Partial<Record<AlertType, WizardRuleTemplate>> = {
```
Comment (scttcper) [inline src/sentry/static/sentry/app/views/alerts/wizard/options.tsx:50]: Partial here feels weird maybe Exclude the one key instead? can go either way i guess

**Review by scttcper — APPROVED** [review]
lookin good

File: src/sentry/static/sentry/app/views/alerts/wizard/options.tsx (hunk 2 above)
Comment (davidenwang) [reply src/sentry/static/sentry/app/views/alerts/wizard/options.tsx:50]: ahh yea forgot about exclude, will change

**Pull request merged**